CatsAndBoats/catseyexi
# Dnc Mythic Fix for Step Bonus

**_I affirm:_**

## What does this pull request do?
Fixed finishing move bonus on steps with Terpsichore equipped

## Steps to test these changes
Job change to DNC.  Equip Terpsichore.  Use Box Step.  It should reward 3 steps instead of 2.
